variables.tf

```
@@ -74,6 +69,23 @@ variable "monitoring_enabled" {
   description = "Enable monitoring for the Auto Scaling Group"
 }
 
+variable "tailscale_api_token" {
```

Let's shorten all variables. We know it's for the Tailscale anyway
```suggestion
variable "api_token" {
```

Also please double check if it's called "Token" in Tailscale admin, so the language is the same everywhere. if not - please rename